app/javascript/controllers/components/App.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+function App () {
+  return (
+    <div>My app is awesome!</div>
```

Comment:
-[ ] Your components folder should include a treating component, kindly make sure to implement this so your app has a greeting component to render the random message